res/values/strings.xml

```diff
@@ -29,7 +29,7 @@
     <string name="emoji">Emoji</string>
     <string name="attachment">Attachment</string>
     <string name="back">Back</string>
-    <string name="verified">Verified</string>
+    <string name="verified">Introduced</string>
```

Comment:
the bare `Introduced` wording that is used if we do not have introducer information is a bit weird - however, that affects only old installations, not new ones. so the issue will fade out 💀

Reply:
about this one, we could consider removing/hiding the entry in such cases, then no weird word is needed, no extra translation work for legacy stuff, and the verified icon is at the top in the title bar and chat name in chat list and inside chat

Reply:
k, we can easily remove the line on android/ios as the icon is in the title. for desktop, things are different as there is no green checkmark in the title bar, for now, we leave the old string in this case and mark it as deprecated